tests/presentation-definition/multi-credential-array.json

```
@@ -0,0 +1,88 @@
+{
+  "id": "32f54163-7166-48f1-93d8-ff217bdb0653",
+  "name": "Identity and Qualifications Verification",
+  "purpose": "We need to verify your identity and qualifications",
+  "format": {
+    "jwt_vp": {
+      "alg": ["EdDSA", "ES256K"]
+    },
+    "jwt_vc": {
+      "alg": ["EdDSA", "ES256K"]
+    },
+    "ldp_vc": {
+      "proof_type": ["Ed25519Signature2018", "EcdsaSecp256k1Signature2019"]
+    },
+    "ldp_vp": {
+      "proof_type": ["Ed25519Signature2018", "EcdsaSecp256k1Signature2019"]
```

NABD: we rarely use secp256k1, secp256r1 is our default ECDSA curve
```suggestion
      "alg": ["EdDSA", "ES256"]
    },
    "jwt_vc": {
      "alg": ["EdDSA", "ES256"]
    },
    "ldp_vc": {
      "proof_type": ["Ed25519Signature2018", "EcdsaSecp256r1Signature2019"]
    },
    "ldp_vp": {
      "proof_type": ["Ed25519Signature2018", "EcdsaSecp256r1Signature2019"]
```